pyproject.toml

```diff
@@ -114,7 +130,7 @@ dependencies = [ # Optional
 #
 # Similar to `dependencies` above, these must be valid existing
 # projects.
```

Comment:
How about this? I'm missing an explanation here, the comment seems at this time somewhat content-less.

```suggestion
# Optional dependencies the project provides. These are commonly 
# referred to as "extras". For a more extensive definition see:
# https://packaging.python.org/en/latest/specifications/dependency-specifiers/#extras
```

Reply:
I don't mind adding this.
Waiting for approval from others before making a commit.

For me personally, this link was more helpful to understand what extras are:
https://pip.pypa.io/en/stable/cli/pip_install/#examples
(point 7.)
plus, it also leads to the specification (that you have provided)